_config.dev.yml

```
@@ -20,3 +20,4 @@ exclude:
   - "_reports/7*"
   - "_reports/8*"
   - "_reports/9*"
+exclude: ['vendor/bundle']
```

YAMLlint (1.37.1)

[error] duplication of key "exclude" in mapping

(key-duplicates)